llm_demo/evaluation/eval_golden.py

```diff
@@ -162,22 +170,22 @@ class EvalData(BaseModel):
     ),
     EvalData(
         category="Insert Ticket",
-        query="I would like to book flight CY 888 departing from SFO on January 1st 2024 at 6am.",
+        query="I would like to book flight CY 922 departing from SFO on 2024-01-01 at 6:38am.",
```

Comment:
In the future, it may be good to refactor these so you do not have to manually update the query